pkgs/top-level/all-packages.nix

```diff
@@ -10906,6 +10906,8 @@ with pkgs;
     withMinimal = false;
   };
 
+  mkbootimage = callPackage ../tools/misc/mkbootimage { };
```

Comment:
We can add this package to pkgs/by-name.
This is because it does not require any arguments to call and does not have a namespace like python-modules.